lib/exec.js

```diff
@@ -11,6 +11,14 @@ module.exports = function exec(script) {
 	script.env = script.env || {};
 
 	var env = objectAssign(process.env, script.env);
+	for (key in env) {
+		if (env.hasOwnProperty(key)) {
+			env[key] = env[key]
+				.replace(/\$\{([^\}]+)\}/g, function (match, p1) {
+					return process.env[p1];
+				});
+		}
+	}
```

Comment:
I have my IDE set to follow the existing indentation on files. Looks like this file was already using tabbed indentation, see: https://raw.githubusercontent.com/benoror/better-npm-run/master/lib/exec.js